iroh-net/src/endpoint.rs

```diff
@@ -771,6 +767,11 @@ impl Endpoint {
         self.msock.conn_type_stream(node_id)
     }
 
+    /// Get the known node addresses which should be persisted.
+    pub fn node_addresses_for_storage(&self) -> Vec<NodeAddr> {
+        self.msock.node_addresses_for_storage()
```

Comment:
This information can already be constructed from the `Endpoint::connection_infos` response.  Maybe it makes more sense to add a `From<ConnectionInfo> for NodeAddr` impl?

Reply:
good idea, done

Reply:
I'm not sure this is the same. The previous code saved only the verified paths of used nodes and all paths of unverified nodes. Now, as far as i can tell, we will store again paths not active/used for nodes with known verified paths

Reply:
@divagant-martian should be fixed, the filtering is happening now at the node layer